src/components/add-transaction-dialog/add-transaction-dialog.html

```diff
@@ -0,0 +1,127 @@
+<button class="mdc-fab material-icons add-transaction__activation" aria-label="Favorite">
```

Comment:
where does BEM block start?

Reply:
it should be called add-transaction-dialog

Reply:
'big plus' button should not be inside dialog component -- but inside root App component (for now, while we do not have drawer)